docker/Dockerfile

```diff
@@ -45,12 +45,15 @@ RUN set -ex \
     mkdir -p "$path"; \
   done
 
-COPY config ./data/config
+COPY config/* ./data/config/
+
+RUN chown -R graylog:graylog ./data
 
```

Comment:
This shouldn't be necessary, see: https://github.com/Graylog2/graylog2-images/blob/2.3/docker/docker-entrypoint.sh#L14-L22